### PR TITLE
fix(TDI-41114): add counter for standalone, improve tests

### PR DESCRIPTION
**Pull request by mbasiuk-talend**
**What is the current behavior?** (You can also link to an open issue here)
No NB_LINE & ERROR_MESSAGE returned for tSnowflakeRow Standalone version
https://jira.talendforge.org/browse/TDI-41114

**What is the new behavior?**
Store NB_LINE & ERROR_MESSAGE  in container. Improve tests.

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [x] The code coverage on new code >75%
- [x] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: components/components-snowflake/components-snowflake-runtime/src/main/java/org/talend/components/snowflake/runtime/SnowflakeRowStandalone.java [hunk 1]
```diff
@@ -46,14 +47,21 @@
     private static final I18nMessages I18N_MESSAGES = GlobalI18N.getI18nMessageProvider()
             .getI18nMessages(SnowflakeRowStandalone.class);
 
+    static final String NB_LINE = "NB_LINE";
+
+    static final String ERROR_MESSAGE = "ERROR_MESSAGE";
+
```
Comment (Voievidko) [inline components/components-snowflake/components-snowflake-runtime/src/main/java/org/talend/components/snowflake/runtime/SnowflakeRowStandalone.java]: `NB_LINE` and `ERROR_MESSAGE` better to make private in my vision

Comment (mbasiuk-talend) [reply components/components-snowflake/components-snowflake-runtime/src/main/java/org/talend/components/snowflake/runtime/SnowflakeRowStandalone.java]: package-private for unit tests.

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/118/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/118/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Ambasiuk%2FTDI-41114-fix_snowflake_row_standalone_nb_line)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/119/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/119/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Ambasiuk%2FTDI-41114-fix_snowflake_row_standalone_nb_line)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/120/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/120/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Ambasiuk%2FTDI-41114-fix_snowflake_row_standalone_nb_line)

**Pull request merged**